studybot.py

```diff
@@ -42,14 +46,21 @@ def setup_database() -> sqlite3.Connection:
     :return: sqlite3.Connection"""
     db_conn = sqlite3.connect('server.db')
     with closing(db_conn.cursor()) as conn:
+        # Create study_time database table.
         table = 'CREATE TABLE IF NOT EXISTS study_time '
         table += '(id integer, minutes integer, server text)'
         conn.execute(table)
         db_conn.commit()
+        # Create live_timer database table.
         table = 'CREATE TABLE IF NOT EXISTS live_timer '
         table += '(id integer, server text, timestamp real)'
         db_conn.execute(table)
         db_conn.commit()
+        # Create github_invites database table.
+        table = 'CREATE TABLE IF NOT EXISTS github_invites'
+        table += '(discord_id integer, github_id integer)'
```

Comment:
Don't use string concat.

Reply:
Consistency within the project. This is how it's done everywhere.